docs/section23/Rule23-19.md

```diff
@@ -0,0 +1,54 @@
+
+# Airside System - Rule 23-19  
+
+**Rule ID:** 23-19  
+**Rule Description:** Systems 5 - 8, the baseline system shall be modeled with preheat coils controlled to a fixed set point 20F less than the design room heating temperature setpoint.  
```

Comment:
@mkarpman If i remember correctly, previously App G used to require preheat in baseline ONLY when the proposed had preheat. Now it looks like all baseline systems 5-8 need to have preheat, irrespective of the proposed. Is that correct?